hazelcast-spring/src/main/resources/hazelcast-spring-3.6.xsd

```diff
@@ -2007,7 +2007,7 @@
 
     <xs:simpleType name="parameterizedValueType">
         <xs:restriction base="xs:string">
-            <xs:pattern value="(\$|\#)\{([^=^:]+)\}"/>
+            <xs:pattern value="[$#]\{([^=^:]+)\}"/>
```

Comment:
do we really need to exclude those chars `^=^:`? If yes, do we need to have some other?